openassessment/tests/test_workflow_batch_update_api.py

```diff
@@ -0,0 +1,545 @@
+"""
+Tests batch ORA workflow update API
+"""
+import datetime
+import logging
+from django.utils import timezone
+from mock import patch
+import pytest
+
+from submissions import api as sub_api
+from openassessment.test_utils import CacheResetTest
+from openassessment.assessment.models import PeerWorkflow
+from openassessment.assessment.api import peer as peer_api
+from openassessment.workflow import api as workflow_api
+from openassessment import workflow_batch_update_api as update_api
+
+logger = logging.getLogger(__name__)
+
+STUDENT_ITEM = {
+    "student_id": "Tim",
+    "course_id": "Demo_Course",
+    "item_id": "item_one",
+    "item_type": "Peer_Submission",
+}
+
+STEPS = ['peer', 'self']
+
+
+class TestWorkflowBatchUpdateAPI(CacheResetTest):
+
+    def test_get_blocked_peer_workflows(self):
+        tim_sub, tim = self._create_student_and_submission("Tim", "Tim's answer")  # pylint: disable=unused-variable
+        self._create_student_and_submission("Miles", "Miles's answer")
+        pat_sub, pat = self._create_student_and_submission("Pat", "Pat's answer")  # pylint: disable=unused-variable
+
+        blocked = update_api.get_blocked_peer_workflows()
+        # we expect 0 blocked submissions as they were just created
+        self.assertEqual(len(blocked), 0)
+
+        # set Tim's submission create_at date to >7 days ago
+        pw_tim = PeerWorkflow.objects.get(student_id=tim["student_id"])
+        pw_tim.created_at = timezone.now() - datetime.timedelta(days=8)
+        pw_tim.save()
+        blocked = update_api.get_blocked_peer_workflows()
+        # we expect 1 blocked submission
+        self.assertEqual(len(blocked), 1)
+
+        # set Pat's submission create_at date to >7 days ago and set completed_at date
+        pw_pat = PeerWorkflow.objects.get(student_id=pat["student_id"])
+        pw_pat.created_at = timezone.now() - datetime.timedelta(days=8)
+        pw_pat.completed_at = timezone.now() - datetime.timedelta(days=2)
+        pw_pat.save()
+        blocked = update_api.get_blocked_peer_workflows()
+        # we still expect 1 blocked submission (Tim)
+        self.assertEqual(len(blocked), 1)
+        self.assertEqual(blocked[0].student_id, 'Tim')
+
+    def test_get_blocked_peer_workflows_for_course(self):
+        # pylint: disable=unused-variable
+        tim_sub, tim = self._create_student_and_submission("Tim", "Tim's answer")
```

Comment:
If you're not using these variables, just don't assign them.
```suggestion
        self._create_student_and_submission("Tim", "Tim's answer")
```

Reply:
they are being used, line #70